schema/deploy/shipping/views.sql

```diff
@@ -13,37 +13,36 @@ begin;
 -- there needs to be a lag between view development and consumers being
 -- updated, copy the view definition into v2 and make changes there.
 
+drop view shipping.reportable_condition_v1;
 create or replace view shipping.reportable_condition_v1 as
 
-    with reportable as (
-        select array_agg(lineage) as lineages
-        from warehouse.organism
-        where details @> '{"report_to_public_health":true}'
-    )
-
     select
         presence_absence.presence_absence_id,
         organism.lineage::text,
         sample.identifier as sample,
-        barcode,
+        sample_id.barcode as sample_barcode,
+        collection_id.barcode as collection_barcode,
         site.identifier as site,
         presence_absence.details->>'reporting_log' as reporting_log,
         sample.details->'_provenance'->>'workbook' as workbook,
         sample.details->'_provenance'->>'sheet' as sheet,
         sample.details->>'sample_origin' as sample_origin,
-        sample.details->>'swab_site' as swab_site
+        sample.details->>'swab_site' as swab_site,
+        encounter.details ->> 'language' as language
 
     from warehouse.presence_absence
     join warehouse.target using (target_id)
     join warehouse.organism using (organism_id)
     join warehouse.sample using (sample_id)
-    join warehouse.identifier on (
-        sample.identifier = cast(identifier.uuid as text))
+    join warehouse.identifier sample_id on (sample.identifier = cast(sample_id.uuid as text))
+    join warehouse.identifier collection_id on (sample.collection_identifier = cast(collection_id.uuid as text))
     left join warehouse.encounter using (encounter_id)
     left join warehouse.site using (site_id)
 
-    where organism.lineage <@ (table reportable)
+    where organism.lineage <@ 'Human_coronavirus.2019'::ltree
     and present
+    -- We can only report on prospective samples
+    and sample.details ->> 'sample_origin' in ('kiosk', 'mail')
```

Comment:
This seems fine but may require more manual updating if we get more sample_origin sites added. A couple other options (not necessarily robust to manual updates):
* exclude all `sample_origin` != '*_retro' (or explicitly list known retros, `sch_retro`, `uwmc_retro`, `hmc_retro`, `nwh_retro`)
* exclude all where `swab_site` is null (I believe this is only for retros) plus those where `sample_origin` == 'es' (environmental sample)

Side note: I see a couple new `sample_origin` labels in there: 'critical' and 'critical_sample'. Not sure what they mean.

Reply:
I'd rather be explicit about what _is_ allowed so we don't accidently leak any results we are not supposed to export.

Reply:
I asked Peter about `critical_sample` on [slack](https://seattle-flu-study.slack.com/archives/CCAEWSFTK/p1584656638083700?thread_ts=1584645092.080500&cid=CCAEWSFTK). They are internal labels for the lab and were supposed to be scrubbed

Reply:
I concur with Jover re: being explicit about what is allowed.

In order not to solely rely on the manifest data being accurate though, I do wonder about making this:

```sql
collection_identifier_set.name in (…)
```

where we list prospective identifier sets. Thoughts?

Reply:
@tsibley So would this be 
```
((collection_identifier_set.name is null or collection_identifier_set.name in (...))
or (sample.details ->> 'sample_origin' in ('kiosk', 'mail')))
```